MCUboot/target/application/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Application
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/mbed-os CACHE INTERNAL "")
+set(EXP_BLE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/mbed-os-experimental-ble-services CACHE INTERNAL "")
+set(MCUBOOT_PATH ${CMAKE_CURRENT_SOURCE_DIR}/mcuboot CACHE INTERNAL "")
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE INTERNAL "")
+set(APP_TARGET application)
+
+include(${MBED_PATH}/tools/cmake/app.cmake)
+
+project(${APP_TARGET})
+
+add_subdirectory(${MBED_PATH})
+add_subdirectory(${EXP_BLE_PATH}/services/FOTA)
+add_subdirectory(${MCUBOOT_PATH}/boot/bootutil/)
+add_subdirectory(${MCUBOOT_PATH}/boot/mbed/)  # Mbed-MCUboot Port
+
+include_directories(mbed-os-experimental-ble-services)
+
+add_executable(${APP_TARGET})
+
+project(${APP_TARGET})
+
+target_sources(${APP_TARGET}
+    PRIVATE
+        source/main.cpp
+        source/BlockDeviceFOTAEventHandler.h
+        source/BlockDeviceFOTAEventHandler.cpp
+        source/PeriodicBlockDeviceEraser.cpp
+        source/PeriodicBlockDeviceEraser.h
+)
+
+target_link_libraries(${APP_TARGET}
+    PUBLIC
+        mbed-os
```

Comment:
Is there a reason to have different indentation ?

Reply:
Oh interesting... I could revert this to the previous style if you'd like. I maintained this style of indentation in the CMakeLists.txt files I wrote for both the mbed-mcuboot port and its examples. It's consistent with that followed in the `mcuboot/boot/bootutil/CMakeLists.txt`. I do notice that the indentation style of `mcuboot/boot/espressif/CMakeLists.txt` seems to match the previous style.